receiver/couchdbreceiver/factory.go

```diff
@@ -28,7 +28,9 @@ import (
 )
 
 const (
-	typeStr = "couchdb"
+	typeStr                   = "couchdb"
+	defaultCollectionInterval = 60 * time.Second
+	defaultHTTPClientTimeout  = 60 * time.Second
```

Comment:
In my opinion, this only complicates the code. We're only using the values once, so just having them inline is fine. I suggest using `time.Minute` in these cases too.

Reply:
updated